generated-platform-project/quarkus-amazon-services/bom/pom.xml

```diff
@@ -161,1617 +161,1617 @@
       <dependency>
         <groupId>software.amazon.awssdk</groupId>
         <artifactId>accessanalyzer</artifactId>
-        <version>2.17.121</version>
+        <version>2.17.127</version>
```

Comment:
BTW, this should probably not be driven by Camel, do you agree @gsmet? It's not Camel's fault though.